client/my-sites/plans-features-main/index.jsx

```diff
@@ -607,7 +608,15 @@ export class PlansFeaturesMain extends Component {
 				<QuerySites siteId={ siteId } />
 				<QuerySitePlans siteId={ siteId } />
 				<HappychatConnection />
-				<div className="plans-features-main__notice" />
+				<PlanNotice
+					visiblePlanNames={ visiblePlans }
```

Comment:
Can we update it to just `visiblePlans`? 

I wish we could just settle on "plan" and stop referring to things with "plan name" or "plan type" when both refer to product store slugs.

Reply:
I wanted to be explicit here because, generally plan refers to the plan object. But I guess that can be communicated by the type.